src/webapp/components/DeliveryTableRow.js

```diff
@@ -83,20 +82,6 @@ const DeliveryTableRow = (props) => {
             <HouseholdSizeChip size={row["Household Size"]} />
           </Box>
         </TableCell>
-        <TableCell component="th" scope="row">
-          {row.slackPermalink ? (
-            <Link
-              href={row.slackPermalink}
-              target="_blank"
-              underline="always"
-              rel="noopener noreferrer"
-            >
-              {row.Code}
-            </Link>
-          ) : (
-            <span>{row.Code}</span>
```

Comment:
hi @caroqliu  - this is awesome! just double-checking here tho - I think we want the code itself to display on the map table, but just not need to be a link. So, it could be just the code text without a link.  Does that make sense?

Reply:
Yep, apologies for taking such a haphazard first pass at this 😅  Should be fixed now. Please take another look when you have a moment!

Reply:
looks good to me! (& thank you @caroqliu !!) @piratefsh look good to you?